src/crypto/equihash.tcc

```diff
@@ -12,18 +12,12 @@ bool DistinctIndices(const FullStepRow<WIDTH>& a, const FullStepRow<WIDTH>& b, s
 {
     std::vector<eh_index> aSrt = a.GetIndices(len, lenIndices);
     std::vector<eh_index> bSrt = b.GetIndices(len, lenIndices);
-
-    std::sort(aSrt.begin(), aSrt.end());
-    std::sort(bSrt.begin(), bSrt.end());
```

Comment:
It doesn't look like GetIndices performs any sorting so I've updated the variable name.